.github/workflows/hostap.yml

```diff
@@ -0,0 +1,93 @@
+name: hostap and wpa-supplicant Tests
+
+on:
+  workflow_call:
```

Comment:
To save compute resources, I would close the PR, change this line to `push:`, then on your branch, get the build working. Once that's done, you can open the PR again with this line changed back. This has the benefit of only running this workflow, which is generally faster.

Reply:
That is a good idea. Thanks I'll do that.